README.md

```diff
@@ -9,7 +9,7 @@ This authenticator is meant to be used with [Universal Authenticator Library](ht
 EOSIO Labs repositories are experimental.  Developers in the community are encouraged to use EOSIO Labs repositories as the basis for code and concepts to incorporate into their applications. Community members are also welcome to contribute and further develop these repositories. Since these repositories are not supported by Block.one, we may not provide responses to issue reports, pull requests, updates to functionality, or other requests from the community, and we encourage the community to take responsibility for these.
 
 ## Supported Environments
-- The Lynx Authenticator is only supported within the Lynx Mobile App Browser
+- The Lynx Authenticator is supported on Lynx Android and Lynx Desktop 2.0+
```

Comment:
Based on #42, this isn't true currently is it?

Reply:
Ah, very true. I was in the mindset of changing branding and docs. It will be true once the other parts are fixed. Ill can revert the readme for now.